add app/design path

webcreative24 opened the pull request:
add "app/design" as magento editable folder

VitaliyBoyko committed:
"Remove contributor section from README.md

webcreative24 committed:
add app/design path

VitaliyBoyko commented on src/main/java/com/magento/idea/magento2plugin/util/magento/IsFileInEditableModuleUtil.java:
```diff
@@ -39,10 +39,12 @@ public static boolean execute(final Project project, final VirtualFile virtualFi
+                        magentoDesignUrl
                 );
             } else {
                 magentoToFolders.add(
```
magentoPathUrl,
magentoDesignUrl

please add here to

VitaliyBoyko committed:
Merge branch 'release-2025.1.0' into 5.4.0-develop-web

VitaliyBoyko committed:
2527: Fix Magento folder handling and update plugin version.

Refactored logic to handle Magento folder paths more robustly, ensuring design paths are added without overwriting existing ones. Updated plugin version to 2025.1.1 and added relevant changelog entry for bug fix #2527. Incremental version changes made in plugin metadata and build properties.

VitaliyBoyko committed:
2527: static fix

MaximGns commented:
would be nice if this fix is released asap

webcreative24 commented:
> MaximGns

You can install the patched version manually by uploading it to PHPStorm
 here you go
 
[PhpStorm Magento 2 Plugin-2025.1.0.zip](https://github.com/user-attachments/files/20189122/PhpStorm.Magento.2.Plugin-2025.1.0.zip)

VitaliyBoyko committed:
Update click action to use specific point coordinates

Modified the `click()` method in `SharedSteps.kt` to include specific point coordinates, ensuring accurate interaction with the 'New Project' button. This change prevents potential misclicks and improves test reliability.

VitaliyBoyko commented:
@webcreative24  could you please sign CLA

webcreative24 commented:
> @webcreative24 could you please sign CLA

done

VitaliyBoyko committed:
Refactor utility classes and fix configuration inconsistencies

Refactored utility classes to improve structure and clarity by adding proper encapsulation and final declarations. Fixed group ID inconsistencies in plugin XML to align with intended usage. Re-enabled commented-out steps in `uitests.yml` for capturing and uploading artifacts on test failures.

VitaliyBoyko committed:
Fix PS.MarkRootGroup registration and update UI test artifacts

Registered PS.MarkRootGroup to correctly add the action, addressing a missing functionality. Changed test artifact capture condition from Windows to Ubuntu for consistency with the testing environment.

VitaliyBoyko committed:
Add step to terminate Chrome processes in UI test workflow

This ensures no lingering Chrome or Chromium processes interfere with test execution, improving reliability. The step is added both before and during test execution for Ubuntu environments.

VitaliyBoyko committed:
Remove redundant Chrome process termination in UI tests

The step to kill Chrome processes was unnecessary and has been removed to simplify the workflow. This change reduces redundancy and ensures the workflow remains focused on essential tasks.

VitaliyBoyko committed:
Fix browser handling and add Chrome termination on Linux

Ensure browsers are properly closed in all scenarios by invoking `closeBrowser()` in the appropriate locations. Additionally, include termination of Chrome processes on Linux systems for better cleanup.

VitaliyBoyko committed:
Fix browser handling and add Chrome termination on Linux

Ensure browsers are properly closed in all scenarios by invoking `closeBrowser()` in the appropriate locations. Additionally, include termination of Chrome processes on Linux systems for better cleanup.

VitaliyBoyko committed:
Disable test artifact capture and upload on failure.

Commented out steps for capturing and uploading test artifacts in the UI test workflow. This change likely aims to streamline the workflow and reduce unnecessary artifact handling during test failures.
